apps/website/src/components/admin/notifications/SendNotificationForm.tsx

```diff
@@ -0,0 +1,297 @@
+              className={classes(
+                `flex flex-wrap gap-4 border-l pl-3`,
+                !isScheduled && "hidden"
+              )}
+            >
+              <LocalDateTimeField
+                showResetButton
+                name="scheduledStartAt"
+                label="Start (Central Time)"
+                required
+              />
+              <LocalDateTimeField
+                showResetButton
+                name="scheduledEndAt"
+                label="End (Central Time)"
+              />
+            </div>
+          </Fieldset>
+
+          {env.NEXT_PUBLIC_FEATURE_NOTIFICATIONS_ADVANCED && (
+            <Fieldset legend="Publication channels">
+              <div className="flex flex-wrap gap-4">
+                <CheckboxField
+                  name="channel"
+                  value="push"
+                  className="rounded-lg border border-white/20 px-2"
+                >
+                  Push
+                </CheckboxField>
+                <CheckboxField
+                  name="channel"
+                  value="discord"
+                  className="rounded-lg border border-white/20 px-2"
+                >
+                  Discord
+                </CheckboxField>
+                <CheckboxField
+                  name="channel"
+                  value="twitter"
+                  className="rounded-lg border border-white/20 px-2"
+                >
+                  Twitter
+                </CheckboxField>
+                <CheckboxField
+                  name="channel"
+                  value="updatesPage"
+                  className="rounded-lg border border-white/20 px-2"
+                >
+                  Updates page
+                </CheckboxField>
+              </div>
+            </Fieldset>
+          )}
+
+          <Button type="submit" className={defaultButtonClasses}>
+            Send notification
+          </Button>
+        </div>
+      </form>
+
+      {env.NEXT_PUBLIC_FEATURE_NOTIFICATIONS_ADVANCED && (
+        <div className="mt-10 flex gap-4 border-t border-black/50 pt-5">
+          <div className="flex-1">
+            <Headline>OpenGraph (Twitter, Facebook etc.)</Headline>
+
+            <div className="overflow-hidden rounded-xl">
+              <PushImageCanvas
+                title={title}
+                text={text}
+                backgroundImageUrl={previewImageUrl}
+                logoImageUrl={logoImage.src}
+              />
+            </div>
+          </div>
+          <div className="flex-1">
+            <Headline>Instagram</Headline>
+
+            <div className="overflow-hidden rounded-xl">
+              <PushImageCanvas
+                title={title}
+                text={text}
+                backgroundImageUrl={previewImageUrl}
+                logoImageUrl={logoImage.src}
+                width={1080}
+                height={1080}
+              />
+            </div>
+          </div>
+        </div>
```

Comment:
Tried to generate some social media images based on the uploaded image and texts. Moved behind the feature flag as it is not done.